src/main/java/com/neuvector/Scanner.java

```diff
@@ -80,7 +81,7 @@ public static ScanRepoReportData scanRegistry(Registry registry, NVScanner nvSca
             }
             String[] cmdArgs = builder.buildForImage(getNVImagePath(nvScanner.getNvScannerImage(), nvScanner.getNvRegistryURL()));
             String[] credentials = {registry.getLoginPassword(), license};
-            reportData = runScan(cmdArgs, nvScanner.getNvMountPath(), credentials);
+            reportData = runScan(cmdArgs, nvScanner, credentials);
```

Comment:
Instead of adding `log` as a parameter, again passing the `nvScanner` object which contains both `mounthPath` and the `log`.